spec.html

```diff
@@ -72,6 +72,10 @@ <h1>Runtime Semantics: PerformPromiseAny ( _iteratorRecord_, _constructor_, _res
         1. ReturnIfAbrupt(_next_).
         1. If _next_ is *false*, then
           1. Set _iteratorRecord_.[[Done]] to *true*.
+          1. Set _remainingElementsCount_.[[Value]] to _remainingElementsCount_.[[Value]] - 1.
+          1. If _remainingElementsCount_.[[Value]] is 0, then
+            1. Let _valuesArray_ be CreateArrayFromList(_values_).
```

Comment:
`_values_` has been renamed to `_errors_`. (And `_valuesArray_` to `_errorsArray_`.)